openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-cmdsender/src/tools/CommandSender/CommandSender.vue

```diff
@@ -774,9 +798,13 @@ export default {
       this.sendDisabled = false
     },
 
-    processCmdResponse(cmd_sent, response) {
+    processCmdResponse(success, targetName, commandName, cmd_sent, response) {
+      // If it was sent from history it's all in targetName, see sendCmd for details
+      if (commandName === undefined) {
+        ;[targetName, commandName] = targetName.split(' ').slice(0, 2)
```

Comment:
Why does this have a leading semicolon?

Reply:
https://prettier.io/docs/en/rationale.html#semicolons